src/main/resources/static/data-mapping.js

```diff
@@ -687,12 +695,12 @@ function visualizeUsrList(usrData) {
         .on('drag', usrItemDragging)
         .on('dragend', usrItemDragend);
     usritem.call(usrItemDrag);
-    positionUsrList(usr_item_data);
+    positionUsrList(usr_item_data, 250, usr_list_y);
 
     function usrItemMouseOver(d) {
         d3.select(this).select('rect').style('fill', 'Orange');
         if (d.shortTexted) {
-            var len = computeTextLength(d3.select(this).select('text'));
+            var len = +computeTextLength(d3.select(this).select('text'));
```

Comment:
no, this was intended, as a safe measure to guarantee the text length is recognized as a number